### PR TITLE
Improved Broadcast

### PR DESCRIPTION
**Pull request by shredlocker**
This PR should not be broken, I hope...

### CODE REVIEW

**Comment by 3lvis** [comment]
I think we should rename BroadcastContainer to just Broadcast, since we don't plan to use a single Broadcast.

File: Sources/Components/Broadcast/Broadcast.swift [hunk 1]
```diff
@@ -4,250 +4,147 @@
...
-    public override init(frame: CGRect = .zero) {
-        super.init(frame: frame)
-        setup()
-    }
-
-    public required init?(coder aDecoder: NSCoder) {
-        super.init(coder: aDecoder)
-        setup()
-    }
+public protocol BroadcastContainerDelegate: class {
```
Comment (3lvis) [inline Sources/Components/Broadcast/Broadcast.swift]: Here still says Container :)

**Comment by 3lvis** [comment]
Epic work, @shredlocker! 👏

**Comment by shredlocker** [comment]
Should I merge this now? I'm not really happy about how it all turned out code wise, it should work fine in the app as it is now

**Comment by 3lvis** [comment]
🚀 🚀 🚀 🚀 🚀

**Pull request merged**